build.mjs

```diff
@@ -0,0 +1,179 @@
+import fs from 'node:fs/promises';
```

Comment:
The logic in this file is similar to that of the deleted gulpfile.

The gulp zip action has been replaced with the archiver dependency..

Reply:
My first thought looking at this file was: this is nice but do we envision (not for this PR) using TypeScript in the future for this library?

Reply:
To me it doesn't really matter, since the build scripts are very small. In addition most IDEs automatically show the type annotations for plain JavaScript code as well (further enhanced by adding the type annotations in jsdoc comments).